src/runtime/manifest-parser.peg

```diff
@@ -257,14 +257,14 @@ Particle
           location: location() // TODO: FIXME Get the locations of the item descriptions.
         } as AstNode.Description;
         item.description.forEach(d => description[d.name] = d.pattern || d.patterns[0]);
-      } else if (item.kind === 'particle-trust-claim-is-tag' || item.kind === 'particle-trust-claim-derives-from') {
+      } else if (item.kind === 'particle-trust-claim' || item.kind === 'particle-trust-claim') {
```

Comment:
This no longer need to check the kind twice, as they are both the same.

Reply:
Done